.github/CODEOWNERS

```diff
@@ -187,7 +190,7 @@
 /core/templates/css/.stylelintrc @Hudda @DubeySandeep
 /tox.ini @Hudda @DubeySandeep
 /scripts/*.py @DubeySandeep
-/scripts/check_e2e_tests_are_captured_in_ci*.py @kevintab95 @DubeySandeep
+/scripts/check_e2e_tests_are_captured_in_ci*.py @DubeySandeep
```

Comment:
Are these permanent changes (ditto for line 179)? Asking as I don't see a TODO comment!

Reply:
Yup it's a permanent change. I'm unassigning myself from all the Automated QA files.

Reply:
Thanks for the info!